src/main/java/com/nirima/jenkins/plugins/docker/DockerComputerLauncher.java

```diff
@@ -57,8 +56,6 @@ private static SSHLauncher getSSHLauncher(InspectContainerResponse detail, Docke
             String host = hostUrl.getHost();
 
             LOGGER.log(Level.INFO, "Creating slave SSH launcher for " + host + ":" + port);
-            
-            PortUtils.waitForPort(host, port);
```

Comment:
IMHO ssh connection itself should be more responsive and should have timeout and retry that we should be able to handle